integrated_channels/blackboard/models.py

```diff
@@ -96,8 +96,6 @@ def __repr__(self):
 class BlackboardEnterpriseCustomerConfiguration(EnterpriseCustomerPluginConfiguration):
     """
     The Enterprise-specific configuration we need for integrating with Blackboard.
-
```

Comment:
while poking around I noticed that this is here when the table in fact __does__ contain pii. Not that it really matters as the system that would handle interpreting the docstring is not in use